src/test/java/org/opensearch/flowframework/util/ApiSpecFetcherTests.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.util;
+
+import org.opensearch.flowframework.exception.ApiSpecParseException;
+import org.opensearch.rest.RestRequest;
```

Comment:
(Style) If you import `RestRequest.Method` you'll have less verbose class definitions later.